app/Http/Middleware/TransformRequestKeys.php

```diff
@@ -25,7 +25,7 @@ public function handle($request, Closure $next)
                 throw ExceptionFactory::snakeCaseRequest();
             }
 
-            $request->request->replace(ArrayKeyTransformer::toSnake($data));
+            $request->replace(ArrayKeyTransformer::toSnake($data));
```

Comment:
jeśli chodzi o samo wybieranie danych z requesta to `$data = $request->getInputSource()->all();` tutaj nie zadziała, przechodzi tylko z ` $request->request->all()` lub ` $request->all()`